build.sbt

```diff
@@ -74,12 +74,25 @@ import build.Dependencies.{
   Sbt1Version
 }
 
+lazy val bloopTask = project
```

Comment:
Do we need to have it in a separate project?

Reply:
moved to backend